config/database.example.yml

```diff
@@ -0,0 +1,50 @@
+# Install the MySQL driver
```

Comment:
Default rails convention for this file is database.yml.sample, so let's change it.